packages/hadron-demo/services/userService.ts

```diff
@@ -1,20 +1,25 @@
 import { User } from '../entity/User';
 import validate from '../entity/validation/validate';
+import { Container } from '@brainhubeu/hadron-core';
+import { bcrypt } from '@brainhubeu/hadron-auth';
```

Comment:
https://github.com/brainhubeu/hadron/pull/21/files#diff-4d2a8ff234a3cb06b4845b2ffe152595R2 in this file You are importing bcrypt from `bcrypt` library, here from `hadron-auth`. It may create some differences if versions of bcrypt will be different in demo and auth.